src/Orleans/Runtime/RequestContext.cs

```diff
@@ -36,13 +32,13 @@ public static class RequestContext
 
         internal const string CALL_CHAIN_REQUEST_CONTEXT_HEADER = "#RC_CCH";
         internal const string E2_E_TRACING_ACTIVITY_ID_HEADER = "#RC_AI";
+        internal const string E2_E_TRACING_LEGACY_ACTIVITY_ID_HEADER = "#L_RC_AI";
         internal const string ORLEANS_REQUEST_CONTEXT_KEY = "#ORL_RC";
```

Comment:
This is no longer used. But no issue, I'll delete it afterwards, merging for now